app/views/providers/submitted_applications/show.html.erb

```diff
@@ -85,7 +85,7 @@
 
     <dl class="govuk-summary-list govuk-summary-list--no-border">
       <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
+        <dt class="govuk-summary-list__key" id="client-signature">
```

Comment:
If you are adding labels to elements purely so that you can style them a particular way, I think it is far better to use a class (so the styling can be reused), and make the name of that class meaningful to its purpose. So here something like a class `"print-fixed-bottom"`

Reply:
Done